plugins/strace.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: strace
+spec:
+  version: "v0.0.1"
+  homepage: https://github.com/michaelwasher/kstrace
+  shortDescription: "Captures strace logs from a running workload in the Kubernetes cluster"
+  description: |
+    kubectl-strace is a kubectl plugin that provides the ability to easily perform debugging of system-calls 
```

Comment:
wrapping this paragraph by 80 characters would look great.

Reply:
Sorry, you must have to say this all the time. I have done this now but can you please confirm... Is it 80 characters including or excluding the starting spaces?

Reply:
excl. the spaces (how it renders in user's terminal) but it's alright :)